src/components/Groups/GroupsPage.tsx

```diff
@@ -141,9 +142,9 @@ const render = props => {
             return (
               <Aux>
                 <ListPane
-                  sortableFields={RESOURCE_MAP.users.sortableFields}
+                  sortableFields={RESOURCE_MAP.users.schema.filter(field => field.sortable)}
                   initialSortOrder={RESOURCE_MAP.users.initialSortOrder}
-                  initialSortField={RESOURCE_MAP.users.initialSortField}
+                  initialSortField={RESOURCE_MAP.users.schema.find(field => field.initialSort)}
```

Comment:
let's revert this, see my pull https://github.com/kids-first/kf-drp-auth-ui/pull/67